### DISCUSSION TITLE
setup.py not found when trying to install in editable mode

### DISCUSSION THREAD

**Discussion by sophondetector** [3b1b/manim#1611] [Q&A] [answered]
when trying to install in editable mode a la the README.md, i get an error saying that the setup.py is required. i note that there does not seem to be a setup.py in the `master` branch.

am i missing something? are contributors generally expected to write their own setup.pys?

thanks

**Comment by TonyCrane** [comment, marked as answer]
Sorry about that, in some cases pip can't install a package under PEP-517 which only has `pyproject.toml` and `setup.cfg` in editable mode. I'll soon add back `setup.py` to fix this. 
Thanks for your feedback.

**Reply by TonyCrane** [reply]
I just added back `setup.py`, can you update your code and try it again? @nateisgood

**Reply by sophondetector** [reply]
@TonyCrane it is working, thank you
